python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -712,9 +742,14 @@ async def do_disableCorrection(self, id_data):
         await asyncio.sleep(0.)  # give control back to event loop
         self.publish_enable_corrections()
 
+    # TODO: This requires implementation
     async def do_setWavelength(self, id_data):
         """Set wavelength to optimize focus.
 
+        Warning
```

Comment:
Warning should come after Parameters, Also, instead of adding "This is not yet implemented" in the docstring, raise a `NotImplementedError` exception. If you want to leave it in the docstring, you could probably just remove the "warning" section.

I would recommend something like this.
```
raise NotImplementedError("This method is not yet implemented. (DM-12345)")
```